crates/router/Cargo.toml

```diff
@@ -26,6 +26,7 @@ vergen = ["router_env/vergen"]
 multiple_mca = ["api_models/multiple_mca"]
 dummy_connector = ["api_models/dummy_connector"]
 external_access_dc = ["dummy_connector"]
+detailed_errors = ["api_models/detailed_errors", "error-stack/serde"]
```

Comment:
Can you add this feature in `integ`, also we would need this in sandbox as well right?

Reply:
As you can see there is not integ feature flag here, and it's not preferred to add it here